src/pages/ReportParticipantsPage.js

```diff
@@ -46,46 +42,8 @@ const defaultProps = {
     personalDetails: {},
 };
 
-/**
- * Returns all the participants in the active report
- *
- * @param {Object} report The active report object
- * @param {Object} personalDetails The personal details of the users
- * @return {Array}
- */
-const getAllParticipants = (report, personalDetails) => {
-    const {participantAccountIDs} = report;
-
-    return _.chain(participantAccountIDs)
-        .map((accountID, index) => {
-            const userPersonalDetail = lodashGet(personalDetails, accountID, {displayName: personalDetails.displayName || 'Hidden', avatar: ''});
-            const userLogin = Str.removeSMSDomain(userPersonalDetail.login || '') || 'Hidden';
-
-            return {
-                alternateText: userLogin,
-                displayName: userPersonalDetail.displayName,
-                accountID: userPersonalDetail.accountID,
-                icons: [
-                    {
-                        id: accountID,
```

Comment:
@youssef-lr  this line was removed in the refractored `getAllParticipants`, i could see this `id` being used in MultipleAvatars component. I wanted to check if this was an intentional change, so that the MultipleAvatars will use the source provided here instead of fetching again by the component? Or if this was a code miss during refractor

https://github.com/Expensify/App/blob/main/src/components/MultipleAvatars.js#L80-L87

Reply:
Good catch! I'll add it back I think I removed it by mistake.